gno.land/cmd/gnoweb/README.md

```diff
@@ -1,13 +0,0 @@
-# gnoweb
```

Comment:
Keep a README in this folder, but feel free to mention that people can view previews on gno.land or run it with gnodev, linking to the gnodev folder.